lib/engine/game/g_1871/step/buy_train.rb

```diff
@@ -59,9 +59,8 @@ def can_sell?(entity, bundle)
             return false unless sellable_bundle?(bundle)
             return false if @game.class::MUST_SELL_IN_BLOCKS && @corporations_sold.include?(bundle.corporation)
 
-            # This is our new clause for 1871, if this is the corporation
-            # selling, we can sell all of them
-            return true if bundle.corporation == bundle.owner
```

Comment:
Does removing this test change anything? Are there any bundles of treasury shares that would be rejected by the `selling_miniumum_shares?` test which should be issuable?

Reply:
good call. Update incoming. 

I'm going to simplify my clause and restore this one above it, only returning false if the bundle.percent > @game.class::TURN_SELL_LIMIT, since the corp sale will already have returned `true` before that check.